autosubmit/job/job_list.py

```diff
@@ -1377,14 +1370,17 @@ def _manage_job_dependencies(self, dic_jobs, job, date_list, member_list, chunk_
             if dic_jobs.as_conf.jobs_data.get(aux_key, {}).get("RUNNING", "once") == "chunk":
                 if aux_key in distances_of_current_section and distance > distances_of_current_section[aux_key]:
                     distances_of_current_section[aux_key] = distance
-            elif dic_jobs.as_conf.jobs_data.get(aux_key, {}).get("RUNNING", "once") == "member":
-                if aux_key in distances_of_current_section_member and distance > distances_of_current_section_member[aux_key]:
-                    distances_of_current_section_member[aux_key] = distance
+
+            elif dic_jobs.as_conf.jobs_data.get(aux_key, { }).get("RUNNING", "once") == "member":
+                if aux_key in distances_of_current_section_member and distance > distances_of_current_section_member[
+                    aux_key]:
+                    distances_of_current_section_member[aux_key]=distance
 
         # Process sections with special filters
-        sections_to_calculate = [key for key in dependencies_keys.keys() if key not in dependencies_to_del]
-        natural_sections = []
+        sections_to_calculate=[key for key in dependencies_keys.keys() if key not in dependencies_to_del]
+        natural_sections=[]
```

Comment:
Missing space. No space in args of functions without types, and spaces everywhere else, I think. Easier to trust IDE/linter. These two can be reverted.